plugins/generator-1.17.1/forge-1.17.1/triggers/bucket_filled.java.ftl

```diff
@@ -0,0 +1,20 @@
+@Mod.EventBusSubscriber public class ${name}Procedure {
+	@SubscribeEvent public static void onBucketFill(FillBucketEvent event) {
+		Player entity=event.getPlayer();
+		double i=entity.getX();

```

Comment:
As we are doing code cleanup, maybe it would make sense to move these eg.

`double i=entity.getX();`

directly into

`dependencies.put("x", entity.getX());`

You may do this for existing templates you added too, so the code we generate becomes a bit cleaner :)